.github/actions/spelling/expect.txt

```diff
@@ -536,6 +544,7 @@ simpleapp
 simplenode
 simplenodeservice
 sitemap
+SJLS
```

Comment:
what is this?

Reply:
I have no idea, that's what the spelling action told me to add :p

Reply:
most likely is matching the GA tag

Reply:
you could fix the with an ignore pattern instead of adding the random string to the expected words :)